docs/ref/webhook/item-update.rst

```diff
@@ -1,2 +1,140 @@
+|                           |               | start time, etc.                                     |
++---------------------------+---------------+------------------------------------------------------+
+| **category**              | *string*      | The name of the category this item is in.            |
++---------------------------+---------------+------------------------------------------------------+
+
+.. _note-item-video:
+
+item.video
+--------------------------------
++---------------------------+---------------+------------------------------------------------------+
+| Field                     | Type          | Description                                          |
++===========================+===============+======================================================+
+| **id**                    | *string*      | This is the identifying string for the Youtube URL.  |
++---------------------------+---------------+------------------------------------------------------+
+| **start**                 | *integer*     | How many seconds into the video to start.            |
++---------------------------+---------------+------------------------------------------------------+
+
+
+.. _note-item-image:
+
+item.image
+---------------------------------
++---------------------------+---------------+------------------------------------------------------+
+| Field                     | Type          | Description                                          |
++===========================+===============+======================================================+
+| **#**                     | *integer*     | The number of the image on the item. The images are  |
+|                           |               | numbered in the order they were uploaded, but ordered|
+|                           |               | in the order they display on the item.               |
++---------------------------+---------------+------------------------------------------------------+
+
+.. _note-item-image-images:
+
+item.image.images
+--------------------------------
++---------------------------+---------------+------------------------------------------------------+
+| Field                     | Type          | Description                                          |
++===========================+===============+======================================================+
+| **src**                   | *string*      | The identifying string for the image.                |
++---------------------------+---------------+------------------------------------------------------+
+| **path**                  | *string*      | The identifying path to where the image is found     |
+|                           |               | relative to your Checkfront Account URL.             |
++---------------------------+---------------+------------------------------------------------------+
+| **url**                   | *string*      | The identifying path to where the image is found     |
```

Comment:
Maybe specify that this is the full sized version, since you specify a size for the following two fields.